conformance-test/libraryMethods.ts

```diff
@@ -187,15 +192,14 @@ export async function bucketUploadResumable(bucket: Bucket) {
     delete bucket.instancePreconditionOpts.ifMetagenerationMatch;
     bucket.instancePreconditionOpts.ifGenerationMatch = 0;
   }
-  createTestFileFromBuffer(
-    FILE_SIZE_BYTES,
-    path.join(__dirname, 'test-data/tmp.dat')
-  );
-  await bucket.upload(path.join(__dirname, 'test-data/tmp.dat'), {
+  const filePath = path.join(__dirname, `test-data/tmp-${uuid.v4()}.dat`);
+  createTestFileFromBuffer(FILE_SIZE_BYTES, filePath);
+  await bucket.upload(filePath, {
     resumable: true,
     chunkSize: CHUNK_SIZE_BYTES,
     metadata: {contentLength: FILE_SIZE_BYTES},
   });
+  deleteTestFile(filePath);
```

Comment:
what happens if the test file is not deleted? do we delete buckets as well? (i dont recall that we do)

Reply:
This is cleaning up the local file not the file in GCS / emulator. I suppose it is unnecessary as everything will get wiped when the environment is torn down but I figured better to just clean it up anyway.